pyquil/quilatom.py

```diff
@@ -537,6 +537,51 @@ def __neg__(self) -> "Mul":
     def _substitute(self, d: Any) -> ExpressionDesignator:
         return self
 
+    def _evaluate(self) -> np.complex128:
+        """
+        Attempts to evaluate the expression to by simplifying it to a complex number.
+
+        Expression simplification can be slow, especially for large recursive expressions.
+        This method will raise a ValueError if the expression cannot be simplified to a complex
+        number.
+        """
+        expr = quil_rs_expr.Expression.parse(str(self))
+        expr.simplify()  # type: ignore
```

Comment:
What's the type error here?

Reply:
For whatever reason, `simplify` doesn't specify a return type in the stubs. Not sure how that got past stubtest. I added the error code to the comment. When it gets fixed upstream, mypy will complain that it's a useless lint.